src/app/users/users-profile/users-profile.component.ts

```diff
@@ -85,13 +85,13 @@ export class UsersProfileComponent implements OnInit {
       this.couchService.put(this.dbName + '/' + userData._id, userData)
     ];
     if (isUserAdmin) {
-      observables.push(this.couchService.get('_users/' + userData._id , { domain: this.userService.getConfig().parentDomain })
+     /*  observables.push(this.couchService.get('_users/' + userData._id , { domain: this.userService.getConfig().parentDomain })
```

Comment:
this is for updating admin password on parent planet. we will have to check if parent planet exist before executing this one to prevent failure